Author: Andreas Hoenen <andreas@hoenen-terstappen.de>
Description: hotfix for BTS report #849679:
Fix display of quotes and nested quotes in greek documents.
--- a/xsl/common/el.xml
+++ b/xsl/common/el.xml
@@ -194,10 +194,10 @@
    <l:gentext key="index symbols" text="Symbols"/>
    <l:gentext key="lowercase.alpha" text="abcdefghijklmnopqrstuvwxyz"/>
    <l:gentext key="uppercase.alpha" text="ABCDEFGHIJKLMNOPQRSTUVWXYZ"/>
-   <l:dingbat key="startquote" text="&amp;#x201C;"/>
-   <l:dingbat key="endquote" text="&amp;#x201D;"/>
-   <l:dingbat key="nestedstartquote" text="&amp;#x2018;"/>
-   <l:dingbat key="nestedendquote" text="&amp;#x2019;"/>
+   <l:dingbat key="startquote" text="&#x201C;"/>
+   <l:dingbat key="endquote" text="&#x201D;"/>
+   <l:dingbat key="nestedstartquote" text="&#x2018;"/>
+   <l:dingbat key="nestedendquote" text="&#x2019;"/>
    <l:dingbat key="singlestartquote" text="&#8216;" lang="en"/>
    <l:dingbat key="singleendquote" text="&#8217;" lang="en"/>
    <l:dingbat key="bullet" text="&amp;#x2022;"/>
